native/src/main/java/io/ballerina/stdlib/crypto/nativeimpl/Encrypt.java

```diff
@@ -34,6 +40,11 @@
  */
 public class Encrypt {
 
+    public static final String COMPRESSION_ALGORITHM = "compressionAlgorithm";
```

Comment:
Since these constants are anyway converted to `BString` its better to use following approach. Same for other constants as well.
```suggestion
    public static final BString COMPRESSION_ALGORITHM = StringUtils.fromString("compressionAlgorithm");
```

And if there constants are not use anywhere other than here, its better to mark them as `private` as well.